chain/network/src/raw.rs

```diff
@@ -0,0 +1,345 @@
+use crate::network_protocol::{
+    Encoding, Handshake, HandshakeFailureReason, PartialEdgeInfo, PeerChainInfoV2, PeerIdOrHash,
+    PeerMessage, Ping, RawRoutedMessage, RoutedMessageBody,
+};
+use crate::time::Utc;
+use bytes::buf::{Buf, BufMut};
+use bytes::BytesMut;
+use near_crypto::{KeyType, SecretKey};
+use near_primitives::block::GenesisId;
+use near_primitives::hash::CryptoHash;
+use near_primitives::network::PeerId;
+use near_primitives::types::{AccountId, BlockHeight, EpochId};
+use near_primitives::version::{ProtocolVersion, PROTOCOL_VERSION};
+use std::io;
+use std::net::SocketAddr;
+use std::time::{Duration, Instant};
```

Comment:
use crate::time instead. It will incur some extra conversions when interacting with tokio API, but that's the direction we are aiming at.